Sources/ProjectDescription/Config.swift

```diff
@@ -38,8 +38,15 @@ public struct Config: Codable, Equatable, Sendable {
     public let plugins: [PluginLocation]
 
     /// Cloud configuration.
+    @available(*, deprecated, message: "Use the `fullHandle` and `url` properties directly in the `Config`")
     public let cloud: Cloud?
 
+    /// The full project handle such as tuist-org/tuist.
+    public let fullHandle: String?
+
+    /// The base URL that points to the Tuist server.
+    public let url: String
```

Comment:
Can't we just get the full URL (e.g. `https://cloud-canary.tuist.io/tuist/tuist-cloud-acceptance-tests`)?

Reply:
Good point. I think that's reasonable. The idea was that by default we use the production server, but this is actually more straightforward. Thoughts @pepicrft?

Reply:
It will make the implementation a bit difficult in a couple of places since we'll need to pluck out the full handle from the URL but it shouldn't be too bad (and we shouldn't drive API decisions based on the internal complexity)

Reply:
I'd keep it separated since the default URL will be the same for most users, so most of the users will only have to pass the full handle.